wordpress-mobile/WordPress-iOS
# Fix wrong "login to see stats" message in Widget

Note: this does not fix yet the new signup case fixed in #15838

Fixes #NA

To test:

- build/run
- install a widget and make sure it has data, or "Unable to load stats" is displayed, in the event that the widget could not load data.
- use the app, navigate through stats and tabs (this should trigger updates on the widgets if you are accessing the same site's stats)
- login and logout a few times and make sure you can either see stats or, if the widget was unable to load them and you are logged in, you see the message "Unable to load stats"
- Even if you get the "Unable to load stats", make sure you can still access your site list from "Edit widget"

PR submission checklist:
